public/js/reader.js

```diff
@@ -163,12 +175,18 @@ Reader.checkFiletypeSupport = function () {
     }
 };
 
-function toastHelpReader() {
-    $.toast().reset("all");
+Reader.toggleHelp = function () {
+    const existingToast = $(".navigation-help-toast:visible");
+    if (existingToast.length) {
+        // ugly hack: this is an abandoned plugin, we should be using something like toastr
+        existingToast.closest(".jq-toast-wrap").find(".close-jq-toast-single").click();
```

Comment:
helpToast.reset() abruptly closes the toast without the fade effect, which is why I opted for a manual close instead, since it's less jarring/better to look at. Storing the toast also means we have to juggle it around the instance. 
By looking [at the code](https://github.com/kamranahmedse/jquery-toast-plugin/blob/9ed0b7ac77bcd77eff79d07c7d340762aa38ffd1/demos/js/jquery.toast.js#L306) doesn't look like there's a way to properly close the toast normally.
Lemme know if you want me to use reset() anyway.

Reply:
I forgot that `reset` didn't play the fade animation!  
Considering that, it's probably better to keep this hack instead and save the effort for an eventual toastr integration.